tests/test_ts.py

```diff
@@ -1,6 +1,7 @@
 from altairexpress import ts
 import pytest
 import altair as alt
+import pandas as pd
 
 
 @pytest.fixture
```

Comment:
You shouldn't have pytest.fixture directly above your testing function because then pytest won't register it as a test. You can check out my test file to see the proper use for it!
```suggestion

```